### PR TITLE
Move !editor to vim_timer

### PR DESCRIPTION
**Pull request by ddellspe**

### CODE REVIEW

File: .activate.sh [hunk 1]
```diff
@@ -1 +1 @@
-venv/bin/activate
\ No newline at end of file
+venv/bin/activate
```
Comment (ddellspe) [inline .activate.sh:1]: this was pre-commit's fault

Comment (asottile) [reply .activate.sh:1]: wellll windows's fault

Comment (ddellspe) [reply .activate.sh]: fixeded

**Review by asottile — APPROVED** [review]

**Pull request merged**